MovieApp/SupportingFiles/Extensions/Int+Extensions.swift

```diff
@@ -0,0 +1,11 @@
+extension Int {
```

Comment:
This is a bit clumsy. We usually create a new struct TimeComponents with multiple constructors like 

- init(minutes: Int)
- init(seconds: Int)
- ....

so that it is more clear what the initial value is.